_config.yml

```diff
@@ -3,4 +3,4 @@ title: Reciprocal Space Station
 plugins:
   - jekyll-font-awesome-sass
 
-port: 4501
```

Comment:
By the way: this changing of the "port" is a silly artifact of how jekyll sites are built locally on my computer and has no bearing on how the site is built on the web. I try to remember to not push these changes, but sometimes I forget. Alas.

Reply:
understood